src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -0,0 +1,12 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class ColorSupplier {
+    private static final String[] COLORS = {"red", "blue", "green", "yellow", "white"};
+
+    public String getRandomColor() {
+        Random random = new Random();
```

Comment:
Since the `Random` instance is not stateful beyond its seed, consider defining it as a private final field of the class instead of creating a new instance every time the `getRandomColor` method is called. This will reduce unnecessary object creation.